service/test/controllers/cache/CacheControllerErrorTest.java

```diff
@@ -0,0 +1,40 @@
+package controllers.cache;
+
+import controllers.BaseApplicationTest;
+import controllers.DummyActor;
+import controllers.DummyErrorActor;
+import modules.OnRequestHandler;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.powermock.core.classloader.annotations.PowerMockIgnore;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.core.classloader.annotations.SuppressStaticInitializationFor;
+import org.powermock.modules.junit4.PowerMockRunner;
+import play.mvc.Http;
+import play.mvc.Result;
+import play.test.Helpers;
+
+@RunWith(PowerMockRunner.class)
+@PrepareForTest({OnRequestHandler.class})
+@SuppressStaticInitializationFor({"util.AuthenticationHelper"})
```

Comment:
no need to suppress static initialization for AuthenticationHelper, please remove @SuppressStaticInitializationFor from all the classes